k8s/environments/prod/common/mi/mi-integration-service-identity.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+ name: mi-integration-service
+ namespace: mi
+spec:
+ type: 0
+ ResourceID: /subscriptions/5ca62022-6aa2-4cee-aaa7-e7536c8d566c/resourceGroups/managed-identities-prod-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/mi-integration-service-prod-mi
```

Comment:
resourceID, clientID, azureIdentity, selector ,This applies to all the identities